sqrt/__init__.py

```diff
@@ -48,6 +49,8 @@ def get_data_stream(iterable):
 
 
 def main(save_to, num_batches, continue_=False):
+    if continue_:
+        continue_training(save_to)
```

Comment:
Oh, and then I meant that you should change the line in this PR to `return continue_training(save_to)`, because right now when `continue_=True` this function first trains the old model, but then creates a new one and trains that one too...